### ISSUE TITLE
Docs accessibility test runner failing

### ISSUE THREAD

**Issue by micahellison** [jrnl-org/jrnl#1932]
### Diagnostic output

n/a

### Current Behavior

Accessibility tests are failing on every PR:
```
Poe => 'tasks:run_shell('"'"'npx pa11y-ci -c config.json'"'"')'
/home/runner/work/jrnl/jrnl/node_modules/puppeteer/lib/cjs/puppeteer/node/BrowserRunner.js:194
            reject(new Error([
                   ^

Error: Failed to launch the browser process!
[1001/013239.500339:FATAL:zygote_host_impl_linux.cc(117)] No usable sandbox! Update your kernel or see https://chromium.googlesource.com/chromium/src/+/master/docs/linux/suid_sandbox_development.md for more information on developing with the SUID sandbox. If you want to live dangerously and need an immediate workaround, you can try using --no-sandbox.
```

### Expected Behavior

The test runner should crawl the site with pa11y-ci for accessibility issues.

### Repro Steps

n/a

### Debug output

n/a
